### PR TITLE
Adds the help/FAQ page and links to it from the dropdown

### LINKED ISSUE #238: Add help/FAQ page
the path should be `https://efolder.cf.ds.va.gov/help`

the content, as provided by @abbyraskin is:
```

<div class="cf-app-segment cf-app-segment--alt cf-help-content">
  <h1>eFolder Express Frequently Asked Questions</h1>

  <ul id="toc" class="usa-unstyled-list">
    <li><a href="#what-is-efolder-express">1. What is eFolder Express?</a></li>
    <li><a href="#what-does-efolder-express-do">2. How was eFolder Express developed? Who was involved?</a></li>
    <li><a href="#how-to-access">3. How do I get access to eFolder Express?</a></li>
    <li><a href="#virtual-va">4. Can I use eFolder Express with Virtual VA documents?</a></li>
    <li><a href="#browser-compatibility">5. Which browser can I use with eFolder Express?</a></li>
    <li><a href="#permission-error">6. After searching for a case, I received the message "You don't have permission to view this eFolder." What does this mean?</a></li>
    <li><a href="#start-retrieving">7. What happens when I click "Start Retrieving eFolder"?</a></li>
    <li><a href="#retrieval-order">8. In what order does eFolder Express retrieve files from VBMS?</a></li>
    <li><a href="#cancel-retrieval">9. Can I cancel the file retrieval process after it starts?</a></li>
    <li><a href="#selecting-documents">10. What if I only need to download specific documents from an eFolder? Can I select the documents I want to download?</a></li>
    <li><a href="#getting-errors">11. Why am I getting errors while trying to retrieve documents from an eFolder?</a></li>
    <li><a href="#download-efolder">12. What happens when I click "Download eFolder"?</a></li>
    <li><a href="#zip-file-name">13. How does eFolder Express name the downloaded eFolder documents?</a></li>
    <li><a href="history">14. What is History? How long does an eFolder remain in my History?</a></li>
    <li><a href="#telecommuting">15. Does eFolder Express work while telecommuting?</a></li>
    <li><a href="#encounter-problems">16. What should I do if I encounter problems?</a></li>
    <li><a href="#share-feedback">17. How do I share my feedback for improving eFolder Express?</a></li>
    <li><a href="#need-help">18. What if I still need help?</a></li>
  </ul>

  <h2 id="what-is-efolder-express">1. What is eFolder Express?</h2>

  <p>
    eFolder Express is a simple web application that lets authorized VA employees bulk-download documents from a veteran's VBMS eFolder, reducing the need to manually click and save these documents one by one. It was built by the Digital Service at VA (DSVA) and will hopefully create cost savings and process improvement for many areas of the VA such as the Office of General Council and the Records Management Center.
  </p>

  <h2 id="what-does-efolder-express-do">2. How was eFolder Express developed? Who was involved?</h2>
  
  <p>
    The Digital Service team worked closely with stakeholders across the VA to develop and test this software over the last several months. Using feedback from VA employees and human-centered design principles, we've tweaked and improved the tool steadily. We will continue to improve the tool  based on feedback and as we have more opportunities to make things simpler.
  </p>

  <h2 id="how-to-access">3. How do I get access to eFolder Express?</h2>

  <p>
    eFolder Express is only accessible to authorized VA employees due to PII and PHI considerations. To get access to eFolder Express, work with your supervisor or ISO to update your permissions. In CSUM/CSEM select "Caseflow" and check the "Download eFolder" box . Employees in station code 101 should use form 20-8824e and select "Caseflow" then "Download eFolder".
  </p>

  <h2 id="virtual-va">4. Can I use eFolder Express with Virtual VA documents?</h2>

  <p>
    No. Right now eFolder Express doesn't support Virtual VA documents. You'll have to download those manually as you have in the past.

  <h2 id="browser-compatibility">5. Which browser can I use with eFolder Express?</h2>

  <p>
    eFolder Express is compatible with modern browsers: Internet Explorer 9 (or later), Firefox, Chrome, and Safari.
  </p>

  <h2 id="permission-error">6. After searching for a case, I received the message "You don't have permission to view this eFolder." What does this mean?</h2>

  <p>
    This usually happens when the eFolder contains sensitive information or when you don't have the privileges required to access it.
  </p>

  <h2 id="start-retrieving">7. What happens when I click "Start Retrieving eFolder"?</h2>

  <p>
    eFolder Express will begin gathering documents from the VBMS database. At this point, you can return to the eFolder Express home page to search for another Veteran ID - eFolder Express is able to retrieve multiple eFolders at the same time. You can also close this browser window while the retrieval is in progress and eFolder Express will still continue gathering the eFolder documents in the background. Remember, you can get back to this page from your History list on the eFolder Express home page.

    It can take anywhere from a few minutes to a few hours to retrieve all the files from an eFolder (depending on its size). Keep in mind, to actually open the eFolder documents, you'll still need to download the eFolder to your computer after the retrieval process completes.
  </p>

  <h2 id="retrieval-order">8. In what order does eFolder Express retrieve files from VBMS?</h2>

  <p>
    eFolder Express retrieves files in order of date received.
  </p>

  <h2 id="cancel-retrieval">9. Can I cancel the file retrieval process after it starts?</h2>

  <p>
    No. Once eFolder Express has started retrieving an eFolder from VBMS you cannot cancel the retrieval. You can, however, choose not to download the eFolder to your computer after it is retrieved. 
  </p>

  <h2 id="selecting-documents">10. What if I only need to download specific documents from an eFolder? Can I select the documents I want to download?</h2>

  <p>
    For now, eFolder Express only supports downloading an entire eFolder. If you only need to download a specific document or set of documents from a veteran's eFolder, you can either download the entire eFolder using eFolder Express and remove the documents you do not need, or you can download the individual documents manually in VBMS.
  </p>

  <h2 id="getting-errors">11. Why am I getting errors while trying to retrieve documents from an eFolder?</h2>

  <p>
    If you are receiving many errors while trying to retrieve an eFolder, it may be because VBMS is down. Because eFolder Express pulls information directly from the VBMS database, if VBMS is not running correctly, eFolder Express will not be able to connect to VBMS to retrieve the eFolder. You can retry the entire eFolder retrieval when VBMS is running again.
    
    You may also encounter errors if your internet connection cuts out temporarily. Again, you will need to start retrieving the entire eFolder again.

    In rare cases there may be a problem with a document in VBMS. If this is the case, you may need to download that document manually from VBMS.
  </p>

  <h2 id="download-efolder">12. What happens when I click "Download eFolder"?</h2>

  <p>
    The browser you are using will ask you to open the folder or to designate a folder where you would like to save the eFolder. VA requires you download the eFolder files to a shared drive rather than your local disk out of consideration for PII/PHI sensitivity. The eFolder you retrieved will then download as a .ZIP file onto your computer with the pattern lastnamefirstname-lastfourdigitsoftheveteransid.zip (for example: doejohn-1234.zip). After you open the .ZIP file you will see the list of downloaded documents in order of receipt date. They will be contained inside a folder labeled in the same manner as the .zip file.
  </p>

  <h2 id="zip-file-name">13. How does eFolder Express name the downloaded eFolder documents?</h2>

  <p>
    Individual documents within an eFolder will be prefixed with a number but otherwise retain their VBMS document type name. Documents will be listed in order of receipt date, and will be saved with the following pattern: documentnumber-documenttype-receiptdate.zip where the date is written 4-digit year, 2-digit month, 2-digit day (for example: 0010-HearingRequest_2016018.zip).
  </p>

  <h2 id="history">14. What is History? How long does an eFolder remain in my History?</h2>

  <p>
    The History section on the eFolder Express home page shows a record of eFolders that you recently retrieved, as well as eFolders the application is currently still retrieving from VBMS. You will see a "View Results" link next to eFolders that have been retrieved from VBMS and are ready to download, and a "View Progress" link next to in-progress eFolder retrievals. If any errors occurred during the retrieval process, you will see a red warning icon next to any eFolder(s) with errors.

    The History list clears automatically after 3 days. 
  </p>

  <h2 id="telecommuting">15. Does eFolder Express work while telecommuting?</h2>

  <p>
    Yes, you can use eFolder Express while connected to the VA network via VPN.
  </p>

  <h2 id="encounter-problems">16. What should I do if I encounter problems?</h2>

  <p>
    If you encounter any problems while using eFolder Express, you should ask your supervisor for assistance. If you are unable to resolve the issue, you can reach the National Service Desk by calling 1-855-NSD-HELP (1-855-673-4357). 
  </p>

  <h2 id="share-feedback">17. How do I share my feedback for improving eFolder Express?</h2>

  <p>
    You can use the "Send feedback" link on the eFolder Express home page.
  </p>

  <h2 id="need-help">18. What if I still need help?</h2>

  <p>
    If you still encounter issues or have questions that aren't answered here, reach out to the National Service Desk by calling
    1-855-NSD-HELP (1-855-673-4357).
  </p>

</div>
```

Make tweaks to the HTML as needed.

### PR DESCRIPTION
**Pull request by amprokop**
Adds a help/FAQ page and links to it.

Fixes #238

Test plan:

- Added automated test to verify that help page appears when "Help" is clicked
- Manual testing

efolder_help_faq

### CODE REVIEW

File: app/views/help/help.html.erb [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<div class="cf-app-segment cf-app-segment--alt cf-help-content">
```
Comment (shanear) [inline app/views/help/help.html.erb]: Are there supposed to be two of these?

File: app/controllers/help_controller.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class HelpController < ApplicationController
+  before_action :authorize
```
Comment (shanear) [inline app/controllers/help_controller.rb]: we don't require authorization for the Certification help and FAQ, i don't think we need to for the efolder either.

Comment (amprokop) [inline app/controllers/help_controller.rb]: Ok, sounds good.

File: app/views/help/help.html.erb (hunk 1 above)
Comment (shanear) [inline app/views/help/help.html.erb]: we could also probably rename this to `show.html.erb` so you could delete this line:
```
render "help"
```

Comment (amprokop) [inline app/views/help/help.html.erb]: okay, i'll try that. thank you for pointing out these rails shortcuts!

**Comment by lakohl** [comment]
@amprokop Yes! Very pretty.

**Comment by lakohl** [comment]
@amprokop Yes! Very pretty.

**Pull request merged**